src/main.cpp

```diff
@@ -42,6 +43,22 @@ int runMixxx(MixxxApplication* app, const CmdlineArgs& args) {
     }
 }
 
+void handleIMVisibleChanged() {
+    if (!QGuiApplication::inputMethod()->isVisible()) {
+        return;
+    }
+    for (QWindow* w : QGuiApplication::allWindows()) {
+        if (std::strcmp(w->metaObject()->className(), "QtVirtualKeyboard::InputView") == 0) {
```

Comment:
normally this should be a qobject_cast, but input view is defined in a private header that can change at any time 
https://github.com/qt/qtvirtualkeyboard/blob/982a9f202780a1f5b3e40f8acf2f9430f18ef35b/src/virtualkeyboard/inputview_p.h
Since it is private even the string presentation is on a risk.

Reply:
What is the risk? If the comparison fails then I think the worst that will happen is the original bug comes back, right?

Reply:
exactly, it's definitely a hack but it's not particularly dangerous